cursorless-talon-dev/src/cursorless_dev.talon

```diff
@@ -26,5 +28,5 @@ tag: user.cursorless
 {user.cursorless_homophone} update cheatsheet:
     user.private_cursorless_cheat_sheet_update_json()
 
-test snippet make <user.cursorless_target>:
-    user.private_cursorless_make_snippet_test(cursorless_target)
+playground <user.cursorless_target>:
+    user.cursorless_command("private.playground", cursorless_target)
```

Comment:
I'd also be tempted to propagate this rename all the way through for consistency